translations/ko/LC_MESSAGES/messages.po

```diff
@@ -39,23 +39,23 @@ msgstr "AMP 블로그"
 
 #: /content/includes/menu.yaml
 msgid "AMP By Example"
-msgstr ""
+msgstr "AMP By Example"
 
 #: /content/includes/stories.yaml
 msgid "AMP Component Library"
-msgstr ""
+msgstr "AMP 구성요소 라이브러리"
 
 #: /views/partials/promo_banner.html:5
 msgid "AMP Conf 2018. Feb 13/14."
-msgstr ""
+msgstr "AMP 컨퍼런스 2018. 2월 13~14일."
 
 #: /views/partials/promo_banner.html:8
 msgid "AMP Conf."
-msgstr ""
+msgstr "AMP 컨퍼런스"
 
 #: /content/includes/menu.yaml
 msgid "AMP Experiments"
-msgstr ""
```

Comment:
AMP 실험실

When translating experiments is '실험'
but in Korea, '실험' is saying some single item.
in this case, It can use '실험실' and it is more natural.